curriculum/updated-curriculum/html/basic-html-06.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <h2>CatPhotoApp</h2>
+    <!-- TODO: Add link to cat photos -->
+    <p>Click here to view more cat photos.</p>
+  </body>
+</html>
+
+<!-- HTML5 has tags that identify different content areas. These tags make your HTML easier to read, and also help with Search Engine Optimization (SEO) and accessibility. Identify the main section of this page by adding a <main> opening tag after the h2 element. Add a </main> closing tag after the p element. -->
```

Comment:
I think at this point we should skip the introduction of more advanced structural elements like "main". We could introduce them in the accessibility project, since they have accessibility implications.